Getting the Response Body/app.js

```diff
@@ -0,0 +1,17 @@
+const https = require("https");
+const request = https.get("https://teamtreehouse.com/chalkers.json", response => {
+	let responseBody = "";
```

Comment:
Always `const` when declaring a variable and the value doesn't change.

Reply:
Thanks, @lycoangelo, This is now updated.